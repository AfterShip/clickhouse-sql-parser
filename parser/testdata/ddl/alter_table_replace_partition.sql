ALTER TABLE t2 REPLACE PARTITION 'partition' FROM t1;