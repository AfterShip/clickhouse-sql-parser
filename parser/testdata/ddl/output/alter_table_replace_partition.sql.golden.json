[
  {
    "AlterPos": 0,
    "StatementEnd": 52,
    "TableIdentifier": {
      "Database": null,
      "Table": {
        "Name": "t2",
        "QuoteType": 1,
        "NamePos": 12,
        "NameEnd": 14
      }
    },
    "OnCluster": null,
    "AlterExprs": [
      {
        "ReplacePos": 15,
        "Partition": {
          "PartitionPos": 23,
          "Expr": {
            "LiteralPos": 34,
            "LiteralEnd": 43,
            "Literal": "partition"
          },
          "ID": null,
          "All": false
        },
        "Table": {
          "Database": null,
          "Table": {
            "Name": "t1",
            "QuoteType": 1,
            "NamePos": 50,
            "NameEnd": 52
          }
        }
      }
    ]
  }
]